CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Improvements:
 
   - Improve Diagnostics: Run linters in parallel #2327
   - Index documents on save #2326
+  - Do not promote parameters that are used in parent constructor #2119
```

Comment:
this is the first feature in a new release now, you can add a new `## master` section :)